src/Pim/Bundle/ApiBundle/Controller/ProductController.php

```diff
@@ -272,9 +272,9 @@ public function createAction(Request $request)
     {
         $data = $this->getDecodedContent($request->getContent());
 
-        $data = $this->populateIdentifierProductValue($data);
```

Comment:
normally you can drop the `populateIdentifierProductValue` method
we have done that in the ProductBuilder if I remember well